gists_cli/actions.py

```diff
@@ -354,28 +377,28 @@ def _getHelpTableRow (action, args='', help=''):
   l.append(util.fileName + ' ' + '|'.join(_cmds[action]) + ' ' + args)
   l.append(help)
   return l
-  
+
 #-------------------------------------------
 
 def help ():
   log.debug ("Help command.")
 
-  print 'Gists.CLI'
-  print ''
-  print textwrap.fill('An easy to use CLI to manage your GitHub Gists. Create, edit, append, view, search and backup your Gists.', defaults.max_width)
-  print ''
-  print 'Author: Nik Khilnani - https://github.com/khilnani/gists.cli'
-  print ''
-  print "Note - GIST_ID can be a Gist ID or Index ID (of the Gist in the List view)."
-  print "Index is 1 based and must be in the format '#N', '%N' , '.N' or ':N'."
+  print('Gists.CLI')
+  print('')
+  print(textwrap.fill('An easy to use CLI to manage your GitHub Gists. Create, edit, append, view, search and backup your Gists.', defaults.max_width))
+  print('')
+  print('Author: Nik Khilnani - https://github.com/khilnani/gists.cli')
+  print('')
```

Comment:
Might be preferable to just add \n to the end of other print statements rather than printing out empty strings